pom.xml

```diff
@@ -89,7 +89,6 @@
 		<module>wicket</module>
 		<module>wicket-core</module>
 		<module>wicket-util</module>
-		<module>wicket-datetime</module>
```

Comment:
So, the module will be completely removed ?!
This is what I suggested some time ago but many devs disagreed.

Reply:
I see no reason to keep empty module :(
AFAICT All valuable staff is moved to wicket-extensions